packages/app-store/salesforce/zod.ts

```diff
@@ -22,7 +22,7 @@ export const appDataSchema = eventTypeAppCardZod.extend({
     .nativeEnum(SalesforceRecordEnum)
     .default(SalesforceRecordEnum.CONTACT)
     .optional(),
-  ifFreeEmailDomainSkipOwnerCheck: z.boolean().optional(),
```

Comment:
Commenting it has a problem that this won't be saved.
I tried using .passthrough() on entire zod object after commenting it, but then it again starts crashing

Reply:
The reason seems to be the big app.schemas.generated.ts having all the apps schemas with different definitions.

Reply:
Found a better fix. using it now.